buildscripts/azure/azure-windows.yml

```diff
@@ -0,0 +1,54 @@
+parameters:
+  name: ''
+  vmImage: ''
+
+jobs:
+- job: ${{ parameters.name }}
+  pool: 
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    maxParallel: 11
+    matrix:
+      py27:
+        PYTHON: '2.7'
+        CONDA_ENV: travisci
+      py37:
+        PYTHON: '3.7'
+        CONDA_ENV: travisci
+
+  steps:
+
+    - powershell: |
+        $wc = New-Object net.webclient
+        $wc.Downloadfile("https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe", "Miniconda3-latest-Windows-x86_64.exe")
+        Start-Process "Miniconda3-latest-Windows-x86_64.exe" "/S /D=C:\Miniconda3" -Wait
+      displayName: 'Install miniconda'
+
+    # Need to install VC 9.0 only for Python 2.7
+    - powershell: |
+        $wc = New-Object net.webclient
+        $wc.Downloadfile("https://download.microsoft.com/download/7/9/6/796EF2E4-801B-4FC4-AB28-B59FBF6D907B/VCForPython27.msi", "VCForPython27.msi")
+        Start-Process "VCForPython27.msi" /qn -Wait
+      displayName: 'Install VC 9.0'
+      condition: eq(variables['PYTHON'], '2.7')
+
+    - script: |
+        call C:\Miniconda3\Scripts\activate.bat
+        call activate base
```

Comment:
why does it needs to be activated twice?

Reply:
Good catch. Left over from all the thrashing before.